src/gc/gcinterface.h

```diff
@@ -41,6 +41,51 @@ typedef enum
     walk_for_loh = 3
 } walk_surv_type;
 
+// Arguments to GCToEEInterface::StompWriteBarrierResize and
+// GCToEEInterface::StompWriteBarrierInitialize. 
+struct WriteBarrierResizeArgs 
```

Comment:
@jkotas  Just to clarify about the flags - should they be indicating to the callback the GC's intent (e.g. the GC indicates it wants StompWriteBarrierResize path and the args struct contains everything needed for that) or the changes that the GC has made (e.g. the GC indicates it has changed the card table, heap high, and heap low pointers and it's up the EE to decide how to react)?